tools/topology/topology1/platform/intel/intel-generic-dmic.m4

```diff
@@ -45,9 +45,10 @@ ifdef(`DMIC16K_PCM_CHANNELS', `', `define(DMIC16K_PCM_CHANNELS, CHANNELS)')
 dnl Unless explicitly specified, dmic period at 48k is 1ms
 ifdef(`DMIC_48k_PERIOD_US', `', `define(DMIC_48k_PERIOD_US, 1000)')
 
-## Prolong period to 4ms for RTNR
-ifdef(`RTNR', `define(`INTEL_GENERIC_DMIC_PERIOD', 4000)', `define(`INTEL_GENERIC_DMIC_PERIOD', 1000)')
-ifdef(`RTNR', `define(`INTEL_GENERIC_DMIC_PERIOD_INV', 250)', `define(`INTEL_GENERIC_DMIC_PERIOD_INV', 1000)')
+## Prolong period to 5ms for RTNR
+ifdef(`RTNR', `define(`DMIC_48k_PERIOD_US', 5000)', `define(`DMIC_48k_PERIOD_US', 1000)')
```

Comment:
This will cause problem for cases of RTNR+AEC on Google projects. DMIC_48k_PERIOD_US is defined as 10000 per AEC requirement under sof-tgl-max98357a-rt5682.m4 which will be replaced by this line.

Please rephrase it or move it to the top-level m4 file (like sof-tgl-max98357a-rt5682.m4) which should be better for keeping this file "generic"

Reply:
Periods settings for RTNR have been moved to top-level m4 file. Please check it, thanks.